tests/e2e/search/search-contacts.wdio-spec.js

```diff
@@ -0,0 +1,77 @@
+const searchPage = require('../../page-objects/search/search.wdio.page');
+const loginPage = require('../../page-objects/login/login.wdio.page');
+const utils = require('../../utils');
+const contactPage = require('../../page-objects/contacts/contacts.wdio.page');
+const commonPage = require('../../page-objects/common/common.wdio.page');
+const placeFactory = require('../../factories/cht/contacts/place');
+const personFactory = require('../../factories/cht/contacts/person');
+const places = placeFactory.generateHierarchy(); // This generates ['district_hospital', 'health_center', 'clinic']
+
+//Add two more health_center
+const sittuHospital = placeFactory.place().build({
+  name: 'Sittu Hospital',
+  type: 'district_hospital',
+  parent: {
```

Comment:
Any reason to have this extra parent chain? Looking at a `district_hospital` created with standard config looks like the first object below. Which is the default on the place factory.

``` 
{
  "parent": "",
}
```
and the whole object for reference. 
```
{
  "_id": "381e39c0-ed9b-4f2d-b2b5-80f2d42443d3",
  "_rev": "2-272bbfe3b659c78eeaabca36e5389981",
  "parent": "",
  "type": "district_hospital",
  "is_name_generated": "true",
  "name": "test's District",
  "external_id": "",
  "notes": "",
  "contact": {
    "_id": "cf03e409-5136-45bb-9667-fbfa7b0dbefd",
    "parent": {
      "_id": "381e39c0-ed9b-4f2d-b2b5-80f2d42443d3"
    }
  },
  "geolocation": "",
  "meta": {
    "created_by": "medic",
    "created_by_person_uuid": "",
    "created_by_place_uuid": ""
  },
  "reported_date": 1628516665637,
  "place_id": "27996"
} ```

Reply:
Done